README.md

```diff
@@ -339,6 +339,14 @@ Sample HTML report:
 
 ![screen](./docs/lighthouse_report.png)
 
+playAudit function also provides a promise that resolves with the Lighthouse result object containing the LHR (Lighthouse report in JSON format).
```

Comment:
can we make the intention more clear for the user? We can also add the type of object returned.

Reply:
I think it is self-explanatory. What would you add?

The important point here is that you can check the report results in the `results` object resolved by the promise returned by playAudit